### ISSUE TITLE
does anybody try to uptrain an exsiting model to obtain a gqa-version model?

### ISSUE THREAD

**Issue by hxs91** [fkodom/grouped-query-attention-pytorch#3]
I'am trying to do this with my own implementation (not with T5), but the initial loss is pretty high (10+), I am not sure whether this is normal.

**Comment by fkodom** [comment]
@hxs91 Have you trained for any significant amount of time?  The initial loss is somewhat arbitrary (dependent on random initialization, training data, etc.), but should go down with additional training steps.

**Comment by hxs91** [comment]
@fkodom yeah, the loss went down with training steps (from 10+ to 5+). But the model weight is not random initialized, the kv head (which is 8 in my setting) is initialized by the mean pooling of original model's kv head, and all other weights is identical to the original model. I originally thought the loss should be lower since it has a good initialization, but it is higher than I expected.
